daq_main2.py

```diff
@@ -28,12 +28,12 @@
 logging.getLogger('ophyd').setLevel(logging.WARN)
 logging.getLogger('caproto').setLevel(logging.WARN)
 handler1 = handlers.RotatingFileHandler('lsdcServerLog.txt', maxBytes=5000000, backupCount=100)
-handler2 = handlers.RotatingFileHandler('/var/log/dama/%slsdcServerLog.txt' % os.environ['BEAMLINE_ID'], maxBytes=5000000, backupCount=100)
+#handler2 = handlers.RotatingFileHandler('/var/log/dama/%slsdcServerLog.txt' % os.environ['BEAMLINE_ID'], maxBytes=5000000, backupCount=100)
```

Comment:
delete stuff you want to remove rather than leaving it commented out.

Reply:
fixed with [b2befa6](https://github.com/NSLS-II/lsdc/pull/315/commits/b2befa6f2f83776ff0ac107083f1f4a2df6223dc)